### DISCUSSION TITLE
How can I add a toggle component in the top nav?

### DISCUSSION THREAD

**Discussion by have-no-clue-what-im-doing** [cloudscape-design/components#1740] [Q&A] [answered]
Trying to find a way to add a Toggle switch in the top navigation component. Not really seeing a way to do that.

**Comment by have-no-clue-what-im-doing** [comment]
I see this here: https://github.com/cloudscape-design/components/discussions/586
Pretty lame if we can't actually add toggles to the top nav. Why even show that on cloudscape.design? It's clearly more intuitive than having the user go to some settings page first each time they want to change it.

**Comment by jperals** [comment, marked as answer]
You are right, such a toggle is currently not supported. To be precise, technically the `search` slot would allow you to put anything you like in the middle of the top nav, but it is not officially supported for something like that, it is meant for an input or autosuggest.

**Reply by have-no-clue-what-im-doing** [reply]
👀
